spec/controllers/build_artifacts_controller_spec.rb

```diff
@@ -10,6 +10,7 @@
     it "should create a build artifact for the build attempt" do
       log_contents = log_file.read
       expect(log_contents).not_to be_empty
+      log_file.rewind
```

Comment:
This resets the `lineno` of the log_file to 0. Without this line, the contents of the build artifact created by the post request below were empty, causing line 20 to fail.